src/main/java/knightminer/inspirations/utility/block/CarpetedPressurePlateBlock.java

```diff
@@ -60,20 +60,24 @@ public String getTranslationKey() {
 		return transKey;
 	}
 
-	// Since fitted carpets may replace the original carpet, we need to lookup the item to make sure we get the right
-	// object. So defer until the first time we actually are pick-blocked.
+	// Since fitted carpets may replace the original carpet, we need to lookup the item to make sure we
+	// get the right object. So defer until the first time we actually need it.
```

Comment:
I thought Forge subs out the entries in Blocks to have the registry subbed variant?

Reply:
It doesn't do the original, but I think `Blocks.BLOCK.delegate.get()` is going to be the correct one. That could be usable, but explicit caching is probably good enough. Well until Forge implements world-specific registry swapping....